addons/l10n_ar/report/invoice_report_view.xml

```diff
@@ -21,7 +21,7 @@
         <field name="name">IIBB - Sales by jurisdiction</field>
         <field name="res_model">account.invoice.report</field>
         <field name="view_mode">pivot</field>
-        <field name="context">{'search_default_current': 1, 'search_default_customer': 1, 'search_default_with_document': 1, 'search_default_groupby_l10n_ar_state_id': 1, 'search_default_groupby_account_id': 1, 'time_ranges': {'field': 'date', 'range': 'this_year'}}</field>
+        <field name="context">{'search_default_current': 1, 'search_default_customer': 1, 'search_default_with_document': 1, 'search_default_company': 1, 'search_default_groupby_l10n_ar_state_id': 1, 'search_default_groupby_account_id': 1, 'time_ranges': {'field': 'date', 'range': 'this_year'}}</field>
```

Comment:
I suppose this is the group by company and we should maybe have given it a better name.

Reply:
@jco-odoo Yes, that's the group by company field from [here](https://github.com/odoo/odoo/blob/65b0338883e173c792ee177eb438d411f214bb8d/addons/account/report/account_invoice_report_view.xml#L90). I do not know if we can, but if you prefer we can change it